internal/rbd/group/util.go

```diff
@@ -197,6 +209,12 @@ func (cvg *commonVolumeGroup) getConnection(ctx context.Context) (*util.ClusterC
 		return cvg.conn, nil
 	}
 
+	if cvg.credentials == nil {
```

Comment:
having this check before calling `cvg.getJournal()` or inside `cvg.getJournal()` function makes sense to me.

Reply:
Good catch, I'll move it to `getJournal()`

Reply:
Actually, it belongs here in `getConnection`, as that uses the credentials too. `getJournal` does not even call `getConnection`. I'll add the check in `getJournal` too.